### PR TITLE
Menu items style

### PR DESCRIPTION
**Pull request by yuval-hazaz**
- [x] MenuItem Component - Use Button instead of Primer SideNav.Link to support menu items without URLS.
- [x] Remove usage of Primer SideNav 
- [x] CommandPalette - Use MenuItem instead of custom JSX
- [x] Remove URL from signout button
- [x] Remove menu collapse to bottom

### CODE REVIEW

**Comment by iddan** [comment]
CI seems to be failing

File: packages/amplication-client/src/Layout/MenuItem.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  border: none !important;
+  box-sizing: border-box;
+  transition: all var(--menu-expand-animation-duration) ease-in;
+  height: auto !important;
+
+  &__title {
+    white-space: nowrap;
+    opacity: 0;
+    transition: all var(--menu-expand-animation-duration) ease-in;
+    @include title1;
```
Comment (iddan) [inline packages/amplication-client/src/Layout/MenuItem.scss:22]: Include should be in the beginning of the block

File: packages/amplication-client/src/CommandPalette/CommandPalette.tsx [hunk 2]
```diff
@@ -97,7 +98,7 @@ const CommandPalette = () => {
 
   return (
     <ReactCommandPalette
-      trigger={<Icon icon="search" />}
+      trigger={<MenuItem title="Search" icon="search" />}
```
Comment (iddan) [inline packages/amplication-client/src/CommandPalette/CommandPalette.tsx:101]: The trigger should come from props.

File: packages/amplication-client/src/Layout/MenuItem.tsx [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
-  const match = useRouteMatch(to);
+  const match = useRouteMatch(to || NON_URL);
+
   return (
-    <SideNav.Link
+    <Button
+      buttonStyle={EnumButtonStyle.Clear}
+      as={to ? NavLink : Button}
       onClick={onClick}
-      as={NavLink}
```
Comment (iddan) [inline packages/amplication-client/src/Layout/MenuItem.tsx:22]: Break the component to one with the `to` prop and one without.

Comment (yuval-hazaz) [reply packages/amplication-client/src/Layout/MenuItem.tsx:22]: I don't see many benefits from splitting this component to two.
We will have to repeat most of the code in both, unless we use two external ones that wrap the same internal component. In this case we will be left with the same internal code - Just two external interfaces. But, I don't think the current interfaces is confusing in any way...

**Pull request merged**